packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -29,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:2.3.2'
+    return 'defi/defichain:master-9e96307'
```

Comment:
```suggestion
    return 'defi/defichain:2.6.0'
```

We can now bump to 2.6.0 since it's released